### DISCUSSION TITLE
BladeBit with PlotMan

### DISCUSSION THREAD

**Discussion by brianwfreeman** [Chia-Network/bladebit#133] [Q&A]
Right now I have 64-core systems pumping out plots in 5-6 minutes using BladeBit; the copy to HDD part takes 12 minutes, so I have a bottle-neck there.  I was thinking of configuring PlotMan to have the plots sent from RAM to a high speed SSD, and then let PlotMan handle the rsync transfer of those plots to the HDDs. 

Has anyone done this successfully?  Are you willing to share your plotman.yaml config file with us to see how you set it up?

**Comment by harold-b** [comment]
I believe @altendky added the archive only ability to do this.

**Reply by altendky** [reply]
I responded over at https://github.com/ericaltendorf/plotman/discussions/938#discussioncomment-1526449.  I'm going to follow up with a hacky option I forgot to mention though.

**Reply by altendky** [reply]
Mmm, actually, I think it's not really applicable here.  You can kind of trick plotman into not seeing other processes' activities, but I think it's not quite enough here.
